bentoml/_internal/server/static_content/index.css

```diff
@@ -1,20 +1 @@
-html {
-    box-sizing: border-box;
-    overflow: -moz-scrollbars-vertical;
-    overflow-y: scroll;
-}
-
-*,
-*:before,
-*:after {
-    box-sizing: inherit;
-}
-
-body {
-    margin: 0;
-    background: #fafafa;
-}
-
-.topbar {
-    display: None;
-}
\ No newline at end of file
+html{box-sizing:border-box;overflow:-moz-scrollbars-vertical;overflow-y:scroll}*,:after,:before{box-sizing:inherit}body{margin:0;background:#fafafa}.swagger-ui .renderedMarkdown code{color:#44a4c6}.swagger-ui .info table{margin:25px 0;font-size:.9em}@supports (font-variation-settings:normal){.swagger-ui .renderedMarkdown code{color:#44a4c6}}.topbar{display:None}.info .main .title{visibility:hidden;position:relative}.info .main .title span{visibility:visible;position:absolute;left:0}.github-corner:hover .octo-arm{animation:560ms ease-in-out octocat-wave}@keyframes octocat-wave{0%,100%{transform:rotate(0)}20%,60%{transform:rotate(-25deg)}40%,80%{transform:rotate(10deg)}}@media (max-width:500px){.github-corner:hover .octo-arm{animation:none}.github-corner .octo-arm{animation:560ms ease-in-out octocat-wave}}.version-section{padding:10px;height:100%}.version-section.dark{background:#151513}
```

Comment:
@aarnphm let's revert this change back... you should never check in minified bundle like thihs